### DISCUSSION TITLE
Statusbar for Flood

### DISCUSSION THREAD

**Discussion by piperun** [jesec/flood#238] [Ideas]
One key feature I'm missing from flood that is present in rutorrent is the bottom statusbar.
![firefox_2021-03-11_02-12-49](https://user-images.githubusercontent.com/1145126/110720714-f95d7b80-8206-11eb-96ee-744fb9287686.png)
It would be nice if things such as: 
- Disk Space
- CPU (optional in my case)
- Memory (optimally showing flood and rtorrent mem usage)
- Port status
- rtorrent/flood version
- Bandwidth usage (optional in my case)

Of course it doesn't have to be at the bottom of the page, it could also be attached at the top (or fill the empty center where the start, stop, add remove torrent buttons are).

If that's not possible, then could "Disk usage" be moved up to where the bandwidth graph is displayed at?
Since it's a bit annoying having to scroll up and down between the two.

**Comment by jesec** [comment]
Those things are in the sidebar. I don’t think a bottom bar is necessary. It doesn’t fit with the design, either. I am open to an additional and optional `available CPU/Memory` row (of the whole system), though. 

> If that's not possible, then could "Disk usage" be moved up to where the bandwidth graph is displayed at?
Since it's a bit annoying having to scroll up and down between the two.

That would be ugly, with the current design. The sidebar would have to be redesigned. Note that regular users typically don’t need to check disk usage frequently.

**Reply by piperun** [reply]
I see, would it at least be possible to make it possible to minimize the different lists (tracker, tags, status) in the sidebar?

**Reply by jesec** [reply]
Good idea.

**Comment by ajhorvath20** [comment]
Is it possible to add port status icon, because I cannot find it how to check if port is open or not. I am running `rtorrent+flood` in docker. Thanks

**Comment by urwrstkn8mare** [comment]
Could there be another icon in the row circled in red like the notifications icon. Except when you hover over this you get some network connection details. For example, connection status, IP, port status and maybe even geographical location from IP.
![image](https://user-images.githubusercontent.com/42769125/163884623-48b87493-e9a6-4f0e-9c4e-53c71b87a2ce.png)
